src/core/trtserver.h

```diff
@@ -792,6 +810,28 @@ TRTSERVER_EXPORT TRTSERVER_Error* TRTSERVER_ServerSharedMemoryAddress(
     TRTSERVER_Server* server, TRTSERVER_SharedMemoryBlock* shared_memory_block,
     size_t offset, size_t byte_size, void** base);
 
+/// \param server The inference server object.
+/// \param shared_memory_block The system shared memory block.
+/// \param offset The offset within the shared memory block to get the
+/// address for.
+/// \param byte_size The size of the memory block within the shared memory
+/// block. Returns error if a block of this size (starting at
+/// 'offset') isn't completely contained in the shared memory block.
+/// \param base Returns the base address.
+/// \return a TRTSERVER_Error indicating success or failure.
+TRTSERVER_EXPORT TRTSERVER_Error* TRTSERVER_ServerCudaSharedMemoryAddress(
```

Comment:
Shouldn't need a new function for this.

Reply:
Removed. Reused  `TRTSERVER_ServerSharedMemoryAddress`